src/server/journal/journal.h

```diff
@@ -34,6 +34,7 @@ class Journal {
 
   LSN GetLsn() const;
 
+  // WHY NO COMMENTS?? :(
```

Comment:
speicifically why this flag was introduced, and whether it's safe to suspend in the general case from custom journal handlers